release-notes/3.0/preview/3.0.0-preview-known-issues.md

```diff
@@ -6,6 +6,7 @@ This document lists known issues for **.NET Core 3.0 Preview releases** which ma
 ## ASP.NET Core
 
 ### Preview 3
+- **The Windows Hosting bundle doesn't exist for Preview3**: The Windows Hosting bundle contained an issue which copied the 32 bit dlls into 64 bit locations. To work aroud this issue, please install the 3.0 Preview 2 Hosting Bundle and run the executable on a command line with the paramaters `OPT_NO_SHAREDFX=1 OPT_NO_RUNTIME=1`. Then install the appropriate .NET and ASP.NET runtimes for preview3.
```

Comment:
Maybe add the hosting bundle link from here or point to Preview 2 download page? https://github.com/dotnet/core/blob/master/release-notes/3.0/preview/3.0.0-preview2-download.md